e2etest/backwardcompatible/patchfile

```diff
@@ -0,0 +1,17 @@
+--- update.go	2022-05-18 11:49:32.937404800 +0300
```

Comment:
I'd name the file according to what it does (not sure we can even drop a comment inside)

Reply:
apparently we can write whatever above the @@